docs/apu_flashing_with_rpi.md

```diff
@@ -0,0 +1,89 @@
+Flashing apu with Raspberry Pi
+==============================
+
+# Preparing environment and building image
+
+Details about building firmware are mentioned in [README](https://github.com/pcengines/apu2-documentation#building-firmware-using-pc-engines-firmware-builder)
+
+# Flashing APU2 using RPi2/3/zero
+
+If You have already built an image with descriptions provided earlier, it is
+time to connect RPi to apu.
+
+1. Prepare wire connections.
+
+* pinout:
+
+| Signal     | RPi Pin number | APU SPI pin header      |
+|------------|----------------|-------------------------|
+| 3V power   | 17  	 		  |	1 VCC (may be omitted)	|
+| GND        | 20      		  |	2 GND				  	|
+| SPI0 CS0   | 24      		  |	3 SPICS					|
+| SPI0 CLK   | 23      		  |	4 SPICLK				|
+| SPI0 MISO  | 21      		  |	5 SPIDI					|
+| SPI0 MOSI  | 19      		  |	6 SPIDO					|
```

Comment:
@miczyg1 this alignment doesn't look correct

Reply:
git vs haroopad...
Fixed.